scripts/changelog-links.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
We don't have a changelog at the moment. Nor are there any releases. I think it is fine if we want to do this and start tagging releases. But please check in the CHANGELOG.md as well.